guides/keras-tuner/distributed_tuning.py

```diff
@@ -0,0 +1,137 @@
+"""
+Title: Distributed Tuning on Multiple GPUs and Machines
```

Comment:
Please switch the title to "Distributed hyperparameter tuning", in the 3 files (we need to make it clearly different from "distributed training")